js/src/components/ContentNavigation.js

```diff
@@ -22,11 +23,12 @@ const ContentNavigation = ({
   const nextIsQuiz = nextLesson && Boolean(nextLesson.questions);
   const nextIsLesson = nextLesson && Boolean(nextLesson.sections);
   const noNextLesson = !sections[currentIndex + 1];
-
+  const noPrevLesson = !sections[currentIndex - 1];
```

Comment:
No need to change but for clarity, in the Future I would call this variable `noPrevSection` because now it is a bit confusing.